pkg/manager/manager_test.go

```diff
@@ -1016,10 +1021,15 @@ var _ = Describe("manger.Manager", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(resp.StatusCode).To(Equal(http.StatusOK))
 
-			// Check liveness path without trailing slash
+			// Check liveness path without trailing slash without redirect
 			livenessEndpoint = fmt.Sprint("http://", listener.Addr().String(), strings.TrimSuffix(defaultLivenessEndpoint, "/"))
```

Comment:
`defaultLivenessEndpoint` no longer ends in `/` so no need to trim it:
```suggestion
			livenessEndpoint = fmt.Sprint("http://", listener.Addr().String(), defaultLivenessEndpoint)
```